models.js

```diff
@@ -17,3 +17,23 @@ exports.fetchArticleByID = (park_id) => {
         return rows[0];
     })
 };
+
+exports.updateArticleById = (article_id, articleInfo) => {
+    return db.query(`SELECT votes FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({rows}) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "Article not found"})
+        } else {
```

Comment:
you don't need to select the article first. What happens if you try to update an article which doesn't exist? Please refactor!

Reply:
Hi David thanks for the feedback.

I introduced the initial article select for those cases where the seeded votes are non-zero. My understanding of the ticket is that I should increment/decrement the vote field by the amount set by the client. So the select allows me to look into the db to do this.

Regarding the error - I get a TypeError - cannot read properties of undefined (reading 'votes'). I'm unclear as to what to do about this part.